### PR TITLE
Add documentation

### LINKED ISSUE #36: Document how to add a new test
And where to find new test cases (example WDL files)

### PR DESCRIPTION
**Pull request by dinithiW**
Fixes #36

### CODE REVIEW

**Review by mr-c — CHANGES_REQUESTED** [review]
How shall we test the resulting CWL to ensure that it runs identically?

Where do we record the original location and license of the WDL files?

What should we do if the test file has a feature that we didn't implement yet? Or otherwise causes a problem?

**Comment by dinithiW** [comment]
> How shall we test the resulting CWL to ensure that it runs identically?
> 
> Where do we record the original location and license of the WDL files?
> 
> What should we do if the test file has a feature that we didn't implement yet? Or otherwise causes a problem?

What is meant by the first point? To make sure that the outputs of the WDL and CWL documents are equivalent?

**Comment by mr-c** [comment]
> What is meant by the first point? To make sure that the outputs of the WDL and CWL documents are equivalent?

Yes, that they both work and produce the same results.

**Comment by dinithiW** [comment]
Modified documentation to address the mentioned points.

File: README.md [hunk 1]
```diff
@@ -30,3 +30,15 @@ program that uses the ANTLR grammar for WDL.
...
+
+### Adding Test Cases
+
+1. Find a WDL workflow. Given below are some links that can be used to find these workflows: 
+    1. https://github.com/broadinstitute/warp
+    2. https://github.com/broadinstitute/viral-pipelines/tree/master/pipes/WDL
+    3. https://github.com/biowdl/tasks
+2. Use the translator to convert the WDL file and save the result to a .cwl file at a specified location. ```python wdl2cwl/main.py  path_to_wdl_file -o specified_location```.
+3. If a problem is encountered during the translation or if the WDL workflow has a feature that has not been implemented yet, submit a new issue with a description of the issue at https://github.com/common-workflow-lab/wdl-cwl-translator/issues
+4. Check whether the outputs of the WDL workflow and the resultant CWL file are equivalent upon giving the required inputs. The WDL workflow can be run using a workflow runner like miniwdl. (Refer the documentation https://github.com/chanzuckerberg/miniwdl) The CWL file can be run using cwltool (Refer the documentation https://github.com/common-workflow-language/cwltool)
```
Comment (mr-c) [inline README.md:42]: Okay, how do we get test data?

Comment (dinithiW) [reply README.md:42]: Actually, I'm not quite sure about this. I haven't been very successful in finding the right files to run the workflows. Files I found through searching online don't always work.

Comment (mr-c) [reply README.md:42]: For tools like Picard they have great tests in the repository for the tool itself: https://github.com/broadinstitute/picard/search?q=CollectQualityYieldMetrics → https://github.com/broadinstitute/picard/blob/master/src/test/java/picard/analysis/CollectQualityYieldMetricsTest.java#L48 → https://github.com/broadinstitute/picard/blob/master/testdata/picard/sam/insert_size_metrics_test.sam

That file is small enough we could check it in to this repo (with attribution) and add an automated test that actually runs the input WDL and the resulting CWL description with this file

**Comment by dinithiW** [comment]
Add how to obtain test data

**Review by mr-c — APPROVED** [review]

**Pull request merged**